docs/ManualDeploymentGuideScalarDLonAWS.md

```diff
@@ -0,0 +1,235 @@
+* You must create subnets with the prefix at least _/24_ for the Kubernetes cluster to work without issues even after scaling. 
+
+### Recommendations
+
+* Kubernetes should be private in production and should provide secure access with SSH or VPN. You can use a bastion server as a host machine for secure access to the private network.
+* Subnets should be in multiple availability zones to achieve fault tolerance for the production use.
+* Kubernetes cluster should have public subnets to enable the public endpoint for envoy LoadBalancer (Scalar DL can be accessed from the internet). The Kubernetes cluster in public is not recommended for production.
+
+_Tip_ 
+
+_If you plan to create three or more nodes in a Kubernetes node group for high availability, create at least 3 subnets in different AZs for the Kubernetes cluster._
+
+[Create an Amazon VPC](https://docs.aws.amazon.com/eks/latest/userguide/create-public-private-vpc.html) with the above requirements and recommendations along with your organizational or application standards.
+
+## Step 2. Set up a database 
+
+In this section, you will set up a database for Scalar DL.
+
+### Requirements
+
+* You must have a database that Scalar DL supports.
+
+Follow the [Set up a database](ScalarDLSupportedDatabase.md) document to set up a database for Scalar DL.
+
+## Step 3. Configure EKS
+
+This section shows how to configure a Kubernetes service for Scalar DL deployment.
+
+### Prerequisites
+
+Install the following tools on your host machine:
+* [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html): In this guide, AWS CLI is used to create a kubeconfig file to access the EKS cluster.
+* [kubectl](https://kubernetes.io/docs/tasks/tools/): Kubernetes command-line tool to manage EKS cluster. Kubectl 1.19 or higher is required.
+
+### Requirements
+
+* You must have an EKS cluster with Kubernetes version 1.19 or above in order to use our most up-to-date configuration files.
+* Kubernetes node group must be labeled with key as `agentpool` and value as `scalardlpool` for [ledger](https://github.com/scalar-labs/scalardl) and [envoy](https://www.envoyproxy.io/) deployment.
+* If you are creating a Kubernetes cluster on a private subnet, you must add a new rule in the Kubernetes Security Group **to enable HTTPS access (Port 443)** from the bastion server to the Kubernetes cluster..
+
+### Recommendations
+
+* Kubernetes node size should be `m5.xlarge` for deploying `ledger` and `envoy` pods, because each node requires 4 vCPUs and 16 GiB of memory for deploying ledger and envoy pods.
+* Node groups should have at least 3 nodes for high availability in production.
+* [Cluster autoscaler](https://docs.aws.amazon.com/eks/latest/userguide/cluster-autoscaler.html) should be configured to adjust the number of nodes in your cluster when pods fail or are rescheduled onto other nodes.
+
+### Create a Kubernetes cluster	
```

Comment:
It has already said `Step 3. Configure EKS` and why repeating the same thing in a different way in all sections?
I think it can be simply like `### Steps`.